pkg/vfs/vfs.go

```diff
@@ -687,10 +699,31 @@ func (v *VFS) Read(ctx Context, ino Ino, buf []byte, off uint64, fh uint64) (n i
 		err = syscall.EBADF
 		return
 	}
+	if h.flags == 0 {
```

Comment:
We need a way to determine if this `fh` is just recovered. 
flag is always equal to 0 because `syscall.O_RDONLY` is 0.